mgmt-lambda/test/handlers/handleUpdate.test.ts

```diff
@@ -192,10 +193,13 @@ describe('Handle mgmt-update', () => {
       })
       .resolves({})
 
-    const result = await handleUpdate(lambdaClient, cloudFrontClient, settings)
-    expect(result.statusCode).toBe(500)
-    const error = JSON.parse(result.body)['error']
-    expect(error).toBe('Lambda function with name fingerprint-pro-lambda-function not found')
+    expect.assertions(6)
+
+    try {
+      await handleUpdate(lambdaClient, cloudFrontClient, settings)
+    } catch (e: any) {
+      expect(e.code).toEqual(ErrorCode.LambdaFunctionNotFound)
+    }
```

Comment:
```suggestion

    await expect(handleUpdate(lambdaClient, cloudFrontClient, settings)).rejects.toThrow(
      expect.objectContaining({
        code: ErrorCode.LambdaFunctionNotFound,
      }),
    )
```

And then we can remove the `expect.assertions` :). Pls also apply it in other places

Reply:
and done 👍